src/parallel.ts

```diff
@@ -15,12 +16,12 @@ export interface ParallelResult<T> {
 }
 
 
-export async function parallel<T>(promises: (parallel.Parallelizable<T>|Promise<T>)[]): Promise<ParallelResult<T>> {
+export async function parallel<T>(promises: (T | parallel.Parallelizable<T> | Promise<T>)[]): Promise<ParallelResult<T>> {
   const wrapper = promises.map(async (p, index) => {
     try {
-      const value: T = isPromise<T>(p)
-        ? await p
-        : await p();
+      const value: T = isFunction(p)
```

Comment:
would feel more comfortable using https://www.npmjs.com/package/lodash.isfunction?activeTab=code, and it's not too heavy

Reply:
We don't currently declare `lodash` as a dependency on `jaasync`. 

Looks like `_.isFunction` is the same implementation:
https://github.com/lodash/lodash/blob/main/src/isFunction.ts

Reply:
suggestion was not including lodash, but including the lodash.isfunction package, hence the link. Which is a more robust impl. Odd it's different than the main `_.isFunction` though